docker/Dockerfile

```diff
@@ -47,23 +47,23 @@ RUN curl https://phar.phpunit.de/phpunit-5.7.5.phar -L -o phpunit.phar \
 	&& mv phpunit.phar /usr/local/bin/phpunit
 
 # Copy a default config file for an apache host
-COPY ./apache_default /etc/apache2/sites-available/000-default.conf
+COPY ./configs/apache_default /etc/apache2/sites-available/000-default.conf
```

Comment:
`configs` (plural) sounds weird to me

Reply:
Fair enough. :-D I guess I was just looking at `docker/logs` and not at `docker/data` while typing it. ;-)

Reply:
Actually it does not sound weird to me, but it sounds weird in English or not used to see a dir named like that. How about `config-templates` ?